config/config.exs

```diff
@@ -157,6 +157,10 @@ config :surface, :components, [
   }
 ]
 
+# Configure Privacy Policies link
+config :oli, :privacy_policies,
```

Comment:
Does a similar change to this need to be made in `config/releases.exs` ?

Reply:
Done 👍